fem/pbilinearform.hpp

```diff
@@ -301,10 +304,18 @@ class ParDiscreteLinearOperator : public DiscreteLinearOperator
    /// Returns the matrix "assembled" on the true dofs
    HypreParMatrix *ParallelAssemble() const;
 
+   /** @brief Returns the matrix assembled on the true dofs, i.e.
+       @a A = P_test^t A_local P_trial, in the format (type id) specified by
+       @a A. */
+   void ParallelAssemble(OperatorHandle &A);
```

Comment:
Maybe I'm missing something, but this seems incorrect -- if we want to have a `ParallelAssemble` for `ParDiscreteLinearOperator` it should be `R_test A_local P_trial` not `P_test^t A_local P_trial`...